src/services/userService.ts

```diff
@@ -1,7 +1,30 @@
 import { type User } from '@prisma/client'
 import { AllIncludeVenue } from '../queries/queries'
 import prisma from '../config/prisma/prisma-client'
+import { PrismaClientKnownRequestError } from '@prisma/client/runtime/library'
 
-export async function getAllUsers(): Promise<User[]> {
-  return prisma.user.findMany(AllIncludeVenue)
+export async function getAllUsers(): Promise<User[] | {}> {
+  const users = prisma.user.findMany(AllIncludeVenue)
+  if (users) {
+    return users
+  } else {
+    return { code: 404, message: 'Cannot find any users in the system.' }
```

Comment:
we need to create one common response object,,, easier to front end.

Reply:
Common response object created, works in all cases atm.